scipy/stats/contingency.py

```diff
@@ -7,9 +7,10 @@
 from functools import reduce
 import numpy as np
 from .stats import power_divergence
+import math
 
 
-__all__ = ['margins', 'expected_freq', 'chi2_contingency']
+__all__ = ["chi2_contingency"]
```

Comment:
this change looks incorrect; `__all__` should contain all public functions in this file

Reply:
I made this change before because adding them to __all__ required that I change the stats.__init__.py reference which listed the modules as 
`
chi2_contingency
contingency.margins
contingency.expected_freq
contingency.association
`
Adding the modules to all seemed to require that I drop the contingency on the last three. I made the change but let me know if I should change it back.

Reply:
The reason is that you added `from .contingency import *`, I think you should undo that change.

Reply:
Still a valid request for change here, I think.